boards/ek-tm4c1294xl/Makefile-app

```diff
@@ -0,0 +1,22 @@
+# Makefile for loading applications onto a Tockloader compatible board

```

Comment:
I think this board isn't tockloader compatible?

Reply:
No, at the moment it is not.